docs/source/software/conda.rst

```diff
@@ -16,14 +19,15 @@ Using a locally installed Conda virtual environment is highly recommended so tha
 You can also have more than one environment with different packages for testing purposes. This procedure uses the Anaconda module already loaded on Discovery.
 
 1. If you are on a login node, move to a compute node by typing ``srun --partition=short --nodes=1 --cpus-per-task=1 --pty /bin/bash``. In the above example, we request for 1 node with 1 cpu core, but you can request for additional resources as per your requirements.
-2. To check what version of Python you have installed, type ``which python``.
-3. To load anaconda, type ``module load anaconda3/2022.01``.
-4. To create your environment, type ``conda create -n <yourenvironmentname> python=3.7 anaconda``, where <yourenvironmentname> is the name you want to give your environment. Tip: to see a list of all of your conda environments, type ``conda info -e``. To save space, also use the ``--prefix=/work/<mygroup>/<mydirectory>`` flag to build in your work directory.
+2. To load anaconda, type ``module load anaconda3/2022.01``.
+3. To check what version of Python you have installed, type ``which python``. With the anaconda module loaded you should see the python directory as /shared/centos7/anaconda3/2022.01/bin/python. If you wish to use a different version of python you can load a python module. For example, ``module load python/3.8.1`` and then confirm that your python directory has changed with ``which python``.
```

Comment:
/shared/centos7/anaconda3/2022.05/bin/python

Reply:
If you wish to use a different version of python you can load a python module. For example, ``module load python/3.8.1`` and then confirm that your python directory has changed with ``which python``.

I am not sure if this is the best practice for using a different python version with anaconda. If you need or want a different version of python I would just create an environment and specify the version of python in that environment

conda create --prefix=/path/to/conda/directory python=3.9 -y

Reply:
Thanks joseph! This is a much better way to set the python version. I thought I tried and it didn't work, but I just tested two versions of python (3.9 and 3.11) and it worked fine. I'll update that here.